awx/main/models/ha.py

```diff
@@ -145,7 +146,19 @@ def get_absolute_url(self, request=None):
 
     @property
     def consumed_capacity(self):
-        return sum(x.task_impact for x in UnifiedJob.objects.filter(execution_node=self.hostname, status__in=('running', 'waiting')))
+        execution_capacity_consumed = 0
+        control_capacity_consumed = 0
+        if self.node_type in ('hybrid', 'execution'):
+            execution_capacity_consumed = sum(
+                x.task_impact for x in UnifiedJob.objects.filter(Q(execution_node=self.hostname) & Q(status__in=('running', 'waiting')))
+            )
+        if self.node_type in ('hybrid', 'control'):
+            control_capacity_consumed = sum(
+                settings.AWX_CONTROL_NODE_TASK_IMPACT
+                for x in UnifiedJob.objects.filter(Q(controller_node=self.hostname) & Q(status__in=('running', 'waiting')))
+            )
+        logger.debug(f"{execution_capacity_consumed} execution capacity and {control_capacity_consumed} control capacity consumed on {self.hostname}")
+        return execution_capacity_consumed + control_capacity_consumed
```

Comment:
Okay, I think this looks valid now. I need to think about performance, but this at least seems correct.